lib/hammer_cli_gutterball/content_report.rb

```diff
@@ -0,0 +1,73 @@
+module HammerCLIGutterball
+  class ContentReportsCommand < Command
+    resource :content_reports
+    module ReportPrintable
+      def adapter
+        :csv
+      end
+
+      def execute
+        if option_async?
+          super
+        else
+          request = send_request
+          task_id = request.is_a?(Hash) ? request['id'] : request
+          task_progress(request)
+          taskoutput = load_task(task_id)
+          if taskoutput['output'] && taskoutput['output']['report_data']
+            print_data(taskoutput['output']['report_data'])
+          end
+          HammerCLI::EX_OK
+        end
+      end
+    end
+
+    class ContentHostStatus < InfoCommand
+      include HammerCLIForemanTasks::Async
+      include ReportPrintable
+
+      action :system_status
+      command_name 'content-host-status'
```

Comment:
was hammer not properly translating `system` to `content-host` ? it's supposed to.